cedar-policy-validator/src/cedar_schema/err.rs

```diff
@@ -401,6 +402,10 @@ pub enum ToJsonSchemaError {
     #[error(transparent)]
     #[diagnostic(transparent)]
     ReservedSchemaKeyword(#[from] ReservedSchemaKeyword),
+    /// Duplicate annotations
+    #[error(transparent)]
+    #[diagnostic(transparent)]
+    DuplicateAnnotations(#[from] DuplicateAnnotations),
```

Comment:
Breaking change. This is a exhaustive public error type. Semver checks is passing, but I think it's not looking into `pub use`'ed items

Reply:
Got it. I guess that we need to move it to a user error thrown by lalrpop, which actually makes the code much simpler.

Reply:
We should track an issue for this breaking change so we remember to apply it for an eventual API version 5.0

Reply:
Implement a non-breaking version.